docs/luigi-client-api.md

```diff
@@ -237,10 +237,25 @@ Returns **[Array](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Gl
 
 -   **since**: 1.4.0
 
+#### addNodeParams
+
+Set node parameters to Luigi Core. The search parameters will be added to the URL.
+
+##### Parameters
+
+-   `searchParams` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** 
```

Comment:
Would need to avoid changing *.md* files  manually. Unless there is an issue, `npm run docu` should automatically change this after the code comments are updated.

Reply:
Yeah, you're right. Thanks for pointing this out.